iquip/cmdtools.py

```diff
@@ -0,0 +1,15 @@
+"""Utility module for handling the command line."""
+
+import subprocess
+
+def run_command(cmd):
```

Comment:
Please annotate the parameter and return types.

Reply:
Oh, I forgot using type hinting! I will update it.